src/collections/_documentation/workflow/notifications/alerts.md

```diff
@@ -3,22 +3,129 @@ title: 'Alerts'
 sidebar_order: 1
 ---
 
-Alerts are configured per project and are based on the rules defined for that project. To modify the rules visit your **Project Settings** > **Alerts** > **Rules** tab. On this page you’ll see a list of all active rules and can add alert rules or modify existing rules.
+Each time you create a new project, or a new issue occurs, Sentry creates a default alert rule. Alerts are generated based upon a project’s alert rules. By default, all project members receive alert notifications via email, and all integrations specified in [actions](#actions) receive alert notifications.
 
-### Conditions
+Alerts can also be configured per-project and are based on the rules defined for that project. To modify the rules visit your **Project Settings > Alerts > Rules > "New Alert Rules" or "Edit Rule"**. In Rules, you’ll see a list of all active rules and can add alert rules or modify existing rules.
 
-Rules provide several conditions that you’re able to configure. These are fairly self-explanatory and range from simple state changes to more complex filters on attributes.
+You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page.
```

Comment:
What do you think of directly linking to project alert settings here? https://sentry.io/settings/account/notifications/alerts/